Appraisals

```diff
@@ -95,7 +95,7 @@ RAILS_VERSIONS = %w(
   4.2.5.1
   4.2.5.2
   4.2.6
-  5.0.0.beta3
+  5.0.0.1
```

Comment:
By the way I think you're gonna have a problem with Rails 5 as it only supports Ruby 2.2. Travis is configured to run tests against Ruby 2.1.